docs/topics/logging.rst

```diff
@@ -170,6 +171,10 @@ listed in `logging's logrecord attributes docs
 <https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior>`_
 respectively.
 
+If :setting:`LOG_TOPLEVEL` is set, then the logs will not display the component
+that prints the log. It is unset by default, hence logs contain the component
```

Comment:
This should be only true for Scrapy components, right? I think it'd be good to mention that.